src/site/content/en/secure/fixing-mixed-content/index.md

```diff
@@ -0,0 +1,232 @@
+  in order to protect users and ensure that all of your content loads.
+tags:
+  - security
+---
+
+Supporting HTTPS for your website is an important step to protecting your site and your users from attack,
+but mixed content can render that protection useless.
+Increasingly insecure mixed content will be blocked by browsers, as explained in [What is mixed content?](/what-is-mixed-content)
+
+In this guide we will demonstrate techniques and tools for fixing existing mixed content issues
+and preventing new ones from happening.
+
+## Finding mixed content by visiting your site
+
+When visiting an HTTPS page in Google Chrome,
+the browser alerts you to mixed content as errors and warnings in the JavaScript console.
+
+In [What is mixed content?](/what-is-mixed-content),
+you can find a number of examples and see how the problems are reported in DevTools.
+
+The example of [passive mixed content](https://passive-mixed-content.glitch.me/) will give the following warnings.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="passive-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when mixed content is detected and upgraded">
+</figure>
+
+[Active mixed content](https://active-mixed-content.glitch.me/) is blocked and displays a warning.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="active-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when active mixed content is blocked">
+</figure>
+
+If you find warnings like these on your site,
+you need to fix the `http://` URLs listed in these errors and warnings in your site's source.
+It's helpful to make a list of these URLs, along with the page you found them on, for use when you fix them.
+
+{% Aside %}
+Mixed content errors and warnings are only shown for the page you are currently viewing,
+and the JavaScript console is cleared every time you navigate to a new  page.
+This means you will have to view every page of your site individually to find these errors.
+{% endAside %}
+
+### Finding mixed content in your site
+
+You can search for mixed content directly in your source code.
+Search for `http://` in your source and look for tags that include HTTP URL attributes.
+Note that having `http://` in the `href` attribute of anchor tags (`<a>`)
+is often not a mixed content issue, with some notable exceptions discussed later.
+
+If your site is published using a content management system
+it is possible that the links to insecure URLs have been inserted into the content when publishing pages.
+For example, images may have been included with a full URL rather than a relative path.
+You will need to find and fix these within the CMS content.
+
+### Fixing mixed content
+
+Once you've found where the mixed content is included in your site's source,
+follow these steps to fix it.
+
+First, find the resource and check to see if it is available securely by changing the `http://` to `https://`.
```

Comment:
I'm not quite clear what's being suggested here.

Does 'find' and 'check' mean opening finding the `http://` URL in source, then opening it in a tab in the browser?

'the Console message says that the resource has been automatically upgraded,' is just another sign that the `https://` resource is available, right?

So maybe, something like:

`If you get a console message that a resource request has been automatically upgraded from HTTP to HTTPS, you can safely change the `http://` URL for the resource in your code to `https://`. You can also check to see if a resource is available securely by changing `http://` to `https://` and attempting to open the URL in a browser tab.`